plugins/modules/handler_info.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2019, Paul Arthur <paul.arthur@flowerysong.com>
```

Comment:
Every file you make is our own, so please remove Paul from here (and from the documentation below)